tests/test_gmail_utils.py

```diff
@@ -0,0 +1,112 @@
+"""
+This is an example automated test to check gmail utils
+Our automated test will do the following:
+    #login to gmail and fetch mailboxes
+    #After fetching the mail box ,select and fetch messages and print the number of messages 
+    #and the subject of the messages
+
+Prerequisites:
+    - Gmail account with app password
+"""
+import sys
+import os
+import pytest
+from integrations.reporting_channels.gmail.gmail import Gmail
+from integrations.reporting_channels.gmail.mailbox import Mailbox
+from dotenv import load_dotenv
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+load_dotenv()

```

Comment:
Are we using load_dotenv()?

Reply:
yes the load_dotenv()  is used to call the .env file,  If load_dotenv() were not used to call, then os.getenv(credentials) will return None.